app/views/text_extraction/_form.html.erb

```diff
@@ -4,13 +4,16 @@
     <% if @cocina_object.type == 'https://cocina.sul.stanford.edu/models/document' %>
        <p>Auto-generating OCR files will not overwrite any existing OCR files that have been corrected to comply with accessibility standards. <strong>Any uncorrected OCR files will be overwritten.</strong> The originally-deposited PDF documents will not be overwritten.</p>
        <p>Avoid auto-generating OCR files for PDF documents that do not contain any text, as it may have adverse effects.</p>
+    <% elsif @cocina_object.type == 'https://cocina.sul.stanford.edu/models/media' %>
+       <p>Auto-generating caption/transcript files will not overwrite any existing caption/transcript files that have been corrected to comply with accessibility standards. <strong>Any uncorrected caption/transcript files will be overwritten.</strong> The originally-deposited media will not be overwritten.</p>
+       <p>Avoid auto-generating caption/transcript files for media that do not contain any speech or lyrics, as it may have adverse effects.</p>
```

Comment:
Maybe replace "may have adverse effects" with "may impact the quality of the resulting caption/transcript".

Reply:
I pulled the language from the design doc: https://docs.google.com/document/d/1ScvlgCI-YhyV2LaaDLTVOWTvuGe4bM1tmgzVbAHTDw8/edit?tab=t.0#bookmark=id.dm5kjr32r5xp  

@alundgard what do you think of Ed's suggested change above? I think it's pretty good.

Reply:
That sounds good to me. The "may have adverse effects" language was borrowed from the OCR messaging, which could error ABBYY depending on the input/parameters.

Reply:
adjusting the language for the caption and will then merge